ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomBaseInput.js

```diff
@@ -32,7 +28,8 @@ const BaseInput = (props) => {
   const classes = styles();
   return (
     <>
-      <div key={props.id} style={style}>
+      <div key={props.id}>
+        <InputLabel htmlFor={props.id}>{prettifiedName}</InputLabel>
```

Comment:
okay, then should I add a condition here on the basis of the prop given maybe? because this is required behavior we need or maybe we can do better your thoughts?